packages/manager/modules/octavia-load-balancer/package.json

```diff
@@ -28,6 +28,7 @@
     "angular": "^1.7.5",
     "angular-translate": "^2.18.1",
     "oclazyload": "^1.1.0",
+    "ovh-api-services": "^16.0.0",
```

Comment:
**niptick:** `ovh-api-services` is deprecated. Can you remove it?
```suggestion
```

Reply:
Done